docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     depends_on:
       jaeger:
         condition: service_started
+      langfuse:
```

Comment:
I would either remove these two lines or comment it out so someone can "force" it to be online, but metrics generally are optional I'd say

Reply:
I commented this line.